cloud/shared/bin/run.py

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env python3
+
+import argparse
+import os
+import sys
+import importlib
+
+sys.path.append(os.getcwd())
```

Comment:
Why is this here between imports? Can you add comment explaining this unconventional call?